tgui/packages/tgui/interfaces/Who.jsx

```diff
@@ -13,18 +13,22 @@ import { Window } from '../layouts';
 export const Who = (props, context) => {
   const { act, data } = useBackend(context);
```

Comment:
useBackend no longer takes an argument.
```suggestion
  const { act, data } = useBackend();
```